### PR TITLE
spellcheck script: ensure that deleted files are not tested

### PR DESCRIPTION
**Pull request by jouvin**
- Also use TRAVIS_COMMIT_RANGE to determine the commits to test

### CODE REVIEW

**Comment by jouvin** [comment]
Attempt to fix the spellchecker problem with deleted files.

**Comment by stdweird** [comment]
@jrha pinging you since this is spellchecker related

File: .travis-scripts/spellchecker [hunk 1]
```diff
@@ -18,4 +18,18 @@ echo
...
+
+# In a pull request, ${TRAVIS_COMMIT_RANGE} contains the commit range of the PR
+# Using it allows to test modified/added files of all PR commits and ensure that
+# deleted files are ignored (git diff provides the summary of all the modifications)
+if [ -z "${TRAVIS_COMMIT_RANGE}" ]
+then
+    commit_range=${TRAVIS_COMMIT_RANGE}
+    echo "Checking modified MD files in commit range ${commit_range}..."
+else
+    oommit_range="HEAD^"
```
Comment (jrha) [inline .travis-scripts/spellchecker:30]: `oommit_range`? Shouldn't this have been `commit_range`?

Comment (jouvin) [reply .travis-scripts/spellchecker:30]: It should... definitely... Strange that it seemed to work once. Minor problem anyway as this branch should not be used anymore in the context of a pull request. I'm opening a PR to fix it.

**Pull request merged**